### ISSUE TITLE
Missing types in wai-aria,  `aria-live` type is mistaking.

### ISSUE THREAD

**Issue by MaxMEllon** [MaxMEllon/kajitsu#36]
https://github.com/MaxMEllon/kajitsu/blob/master/packages/lemon/src/wai-aria.ts#L30
